fontforge/splinestroke.c

```diff
@@ -1074,6 +1074,7 @@ int GenStrokeTracePoints(void *vinfo, bigreal t_fm, bigreal t_to,
 		                                      nib_ccw, CUSPD_MARGIN,
 						      stip->starts_on_cusp);
 		free(fp);
+		*fpp = NULL;
```

Comment:
This gets freed in a different branch below. Why wouldn't we free it here if unsetting it?

Reply:
It might be worth adding an in-line comment on the general logic.

Reply:
`GenStrokeTracePoints()` has two callers. One is the actual tracing function, which has logic based on the number of points returned, with zero points indicating there has been some error condition and there's nothing to be traced for now). The other caller is a level up, which uses the function to find the edges of cusps. 

These changes bring the treatment of the passed memory pointer in line with that semantic, setting it to NULL in the case no memory was allocated. That lets callers who don't care about the number of points to just free the value. 

I'll add some comments to this effect.

Reply:
It could be clearer to just unconditionally null *fpp at the top of the function